cucumber/authenticators_jwt/features/support/authn_jwt_helper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Utility methods for JWT authenticator
+
+module AuthnJwtHelper
+  include AuthenticatorHelpers
+
+  ACCOUNT = 'cucumber'.freeze
+  SERVICE_ID = 'raw'.freeze
+
+  def authenticate_jwt_token(jwt_token)
+    path = "#{conjur_hostname}/authn-jwt/#{SERVICE_ID}/#{ACCOUNT}/authenticate"
+
+    payload = {}
+    payload["jwt"] = jwt_token
```

Comment:
Can we combine this two lines to one lines of
` payload = {"jwt" => jwt_token}`
 ?